native/src/lib.rs

```diff
@@ -529,9 +529,9 @@ fn bbs_get_unblinded_signature(mut cx: FunctionContext) -> JsResult<JsArrayBuffe
 /// {
 ///     "signature": ArrayBuffer,               // The signature to be proved
 ///     "publicKey": ArrayBuffer,               // The public key of the signer
-///     "messages": [String, String]            // All messages that were signed in the order they correspond to the generators in the public key. They will be SHAKE-256 hashed
+///     "messages": [ArrayBuffer, ArrayBuffer]  // All messages that were signed in the order they correspond to the generators in the public key. They will be Blake2b hashed
 ///     "revealed": [Number, Number]            // The zero based indices to the generators in the public key for the messages to be revealed. All other messages will be hidden from the verifier.
-///     "nonce": String                         // This is an optional nonce from the verifier and will be used in the proof of committed messages if present. It is strongly recommend that this be used.
+///     "nonce": ArrayBuffer                    // This is an optional nonce from the verifier and will be used in the proof of committed messages if present. It is strongly recommend that this be used.
```

Comment:
Is there any detail we can add about the consequences of when it's not used given it's "strongly recommend"(ed) to be used?

Reply:
Yes, I'm just not sure this is the best place to put it since the consumers are node developers.

Reply:
Hmm good point, maybe it's best to put it in the documents section and then link in the comment to the docs in both the rust and node comment?